src/lib/utilities/Toast/types.ts

```diff
@@ -21,6 +21,8 @@ export interface ToastSettings {
 	};
 	/** Provide arbitrary CSS classes to style the toast. */
 	classes?: string;
+	/** Callback function that fires on trigger and close. */
+	callback?: (response: { id: string; status: string }) => void;
```

Comment:
Small change since the status can only be either "closed" or "queued" so we can make it a union type.
```suggestion
	callback?: (response: { id: string; status: 'queued' | 'closed' }) => void;
```